File: Atcoder_#332/ruby/a_online_shopping.rb
n, s, k = gets.split(" ").map(&:to_i)

sum = 0
n.times do |i|
  p,q = gets.split(" ").map(&:to_i)
  sum += p * q
end

if sum < s
  puts sum + k
else
  puts sum
end

File: Atcoder_#332/javascript/a_online_shopping.js
function main(input) {
  let lines = input.split('\n');
  let [n, s, k] = lines[0].split(' ').map(Number);

  let sum = 0;
  for (let i = 1; i <= n; i++) {
      let [p, q] = lines[i].split(' ').map(Number);
      sum += p * q;
  }

  if (sum < s) {
      console.log(sum + k);
  } else {
      console.log(sum);
  }
}

main(require('fs').readFileSync('/dev/stdin', 'utf8'));
